scripts/ci/check_compliance.py

```diff
@@ -1060,6 +1060,28 @@ def run(self):
                 self.failure(failure)
 
 
+class BinaryFiles(ComplianceTest):
+    """
+    Check that the diff contains no binary files.
+    """
+    name = "BinaryFiles"
+    doc = "No binary files allowed."
+    path_hint = "<git-top>"
+
+    def run(self):
+        BINARY_ALLOW_PATHS = ("doc/", "boards/")
+        BINARY_ALLOW_EXT = ("jpg", "jpeg", "png", "svg")
```

Comment:
`svg` is not a binary file.

Reply:
Some `svg` in the repo triggers the binary file detection in git for some reason, see my other comment (Gerard pointed it out as well).